src/pages/ImportData.js

```diff
@@ -392,7 +393,27 @@ const ImportData = () => {
     };
   };
 
-  const importTdrExport = (namespace, name) => {
+  const loadWdsUrl = useCallback((workspaceId) => {
```

Comment:
A lot of copied code from `Data.js`.  There's just enough difference that I couldn't just export it as-is; could use some refactoring to make it reuseable.

Reply:
I think it's worth refactoring, unfortunately :(